packages/store/src/store.ts

```diff
@@ -93,6 +101,23 @@ export class Store {
     return selectorFn(this._stateStream.getValue());
   }
 
+  /**
+   * Select a signal from the state.
+   */
+  selectSignal<T>(
+    selector: (state: any, ...states: any[]) => T,
+    options?: CreateComputedOptions<T>
+  ): Signal<T>;
+  selectSignal<T = any>(
+    selector: string | Type<any>,
+    options?: CreateComputedOptions<T>
+  ): Signal<T>;
+  selectSignal<T>(selector: StateToken<T>, options?: CreateComputedOptions<T>): Signal<T>;
```

Comment:
Because immutability is a fundamental premise for data to exist in our state, a user should never have a reason to specify the equality comparison function (which is the only thing that the `CreateComputedOptions` interface provides). 
We should therefore not expose the `options` as an argument to this function.